02/app.js

```diff
@@ -1 +1,22 @@
-console.log('DOM');
\ No newline at end of file
+console.log('DOM');
+
+const links = document.querySelectorAll('a[data-url]')
+
+if (links.length > 0) {
```

Comment:
Tutaj natomiast nie trzeba, bo zawsze jest to tablica. Jeśli nie wyszukano to jest ona pusta więc `for` czy `forEach` się nie wykona, ale błędu nie będzie.